benchmarks/fixed_pool_stress_test.cpp

```diff
@@ -0,0 +1,149 @@
+#include <iostream>
+#include <chrono>
+#include <string>
+#include <random>
+
+#include "umpire/config.hpp"
+
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Allocator.hpp"
+#include "umpire/strategy/FixedPool.hpp"
+
+#define CONVERT 1000000 //convert sec (s) to microsec (us)
+
+const int NUM_RND = 1000; //number of rounds (used to average timing)
```

Comment:
I notice we the use of both `=` and `{}` throughout the code.  Could we perhaps only use `=` where it is required and otherwise always use `{}`?

```suggestion
const int NUM_RND{ 1000 }; //number of rounds (used to average timing)
```

Reply:
Good idea